src/app/zap-templates/common/ClustersHelper.js

```diff
@@ -560,7 +560,7 @@ class ClusterStructUsage {
   }
 }
 
-Clusters._addUsedStructureNames = async function(clusterName, startType, allKnownStructs) {
+Clusters._addUsedStructureNames = async function (clusterName, startType, allKnownStructs) {
```

Comment:
I can't reproduce these changes by running `restyle_diff.sh` locally, so far.  What exact commands were run, starting with a clean master, to generate this diff?

Reply:
I believe js files are already restyled, however maybe only restyled when changed.

I also could not see this style change, only a change in items.forEach invocation in inlineStructItesms